bindata/v4.1.0/alerts/cpu-utilization.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: cpu-utilization
+  namespace: openshift-kube-apiserver
+  annotations:
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+    exclude.release.openshift.io/internal-openshift-hosted: "true"
+spec:
+  groups:
+    - name: control-plane-cpu-utilization
+      rules:
+        - alert: HighOverallControlPlaneCPU
+          summary: >
+            CPU utilization across all three control plane nodes is higher than two control plane nodes can sustain; a single control plane node outage may
+            cause a cascading failure; increase available CPU.
+          message: >
+            Given three control plane nodes, the overall CPU utilization may only be about 2/3 of all available capacity.
+            This is because if a single control plane node fails, the remaining two must handle the load of the cluster in order to be HA.
+            If the cluster is using more than 2/3 of all capacity, if one control plane node fails, the remaining two are likely to
+            fail when they take the load.
+            To fix this, increase the CPU and memory on your control plane nodes.
+          expr: |
+            sum(
+              100 - (avg by (instance) (rate(node_cpu_seconds_total{mode="idle"}[5m])) * 100)
+              AND on (instance) label_replace( kube_node_role{role="master"}, "instance", "$1", "node", "(.+)" )
+            )
+            /
+            count(kube_node_role{role="master"})
+            > 60
```

Comment:
I don't understand the formular. @lilic does it make sense?

Why 60 and not 66.6... ?

do we have an idea of the number on existing customer clusters?

Reply:
In other words, will 80% of customer clusters have the alert after we ship this?

Reply:
> In other words, will 80% of customer clusters have the alert after we ship this?

we don't know.  But it's a warning.